wemake_python_styleguide/visitors/ast/comparisons.py

```diff
@@ -123,3 +124,29 @@ def visit_Compare(self, node: ast.Compare) -> None:
         """
         self._check_ordering(node)
         self.generic_visit(node)
+
+
+class MultipleInVisitor(BaseNodeVisitor):
```

Comment:
Can you please drop here a `# TODO(@sobolevn): refactor to be a single visitor` for me to do later?

Reply:
Added.